src/content/docs/develop/actions.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Actions
+sidebar:
+  order: 1
+---
+
+
+## Contents
+- Cross platform compilation and automating releases
+- Compiling for arm using [Pguyots](https://github.com/pguyot) arm-runner
+- Compiling on self hosted runners
+
+### Basic Cross Platform Compilation
+
+```yaml
+name: 'publish'
+
+on:
+  push:
+    branches:
+      - release
+
+jobs:
+  publish-tauri:
+    permissions:
+      contents: write
+    strategy:
+      fail-fast: false
+      matrix:
+        settings:
```

Comment:
Can you change this settings to include?
reference: https://github.com/tauri-apps/tauri-action/commit/e5c4e13a7c979603440eddb183b266170613b1ba / https://github.com/tauri-apps/tauri-action/pull/772

It appears that `settings` should not be used this way according to the GitHub actions documentation and intellisense.
![image](https://github.com/tauri-apps/tauri-docs/assets/59066707/9613c253-3589-41a5-8ec0-8e688eed986c)

Reply:
Works fine on my repo

Reply:
I'll change it to match the proper usage case.